Volume/Views/Publications/PublicationList.swift

```diff
@@ -11,6 +11,7 @@ import SwiftUI
 
 struct PublicationList: View {
     @State private var cancellableQuery: AnyCancellable?
```

Comment:
you have a linebreak in your `mainView` struct here but i'm not sure which one's standard practice for swiftUI

Reply:
Yea honestly I'm not sure either, but I'm going to start adding a line before because it looks cooler and is easier to read 😎